docs/en/enterprise-edition/content-collections/runtime-security/agentless-scanning/agentless-scanning-modes.adoc

```diff
@@ -80,3 +80,27 @@ This is the default mode to help you get started as soon as you complete onboard
 |Additional configuration required for each account after you complete onboarding your accounts.
 
 |===
+
+[#viewing-accounts-by-scanning-modes]
+[.task]
+=== Viewing Scanning Mode for the Cloud Accounts
+All the cloud accounts appear in the *Runtime Security > Manage > Cloud accounts* page. 
```

Comment:
either remove or edit "to view all cloud accounts navigate to..."

Reply:
I have edited the sentence to "To add the scan mode column for the accounts table, complete the following steps:"